smartsim/database/orchestrator.py

```diff
@@ -132,32 +133,32 @@ def create_cluster(self):  # cov-wlm
         self.check_cluster_status()
         logger.info(f"Database cluster created with {str(len(self.hosts))} shards")
 
-    def check_cluster_status(self):  # cov-wlm
+    def check_cluster_status(self):
```

Comment:
Just a question, why did we remove ` # cov-wlm`?
The reason why we had it was that this is a function which can only be tested when running on WLM, did this change (i.e. can we call this function when running locally)?

Reply:
@ctandon11 ?

Reply:
@al-rigazzi @Spartee you are right, this function still cannot be run locally. I can add `# cov-wlm` back in.